src/Run.js

```diff
@@ -37,8 +42,15 @@ function Run(ref, json_q, callback,ind, progressObj) {
       // Tell UI that a step is being drawn.
       ref.images[image].steps[i].UI.onDraw(ref.images[image].steps[i].options.step);
 
+      var inputForNextStep = ref.copy(input);
+      inputForNextStep.getPixels = getPixels;
+      inputForNextStep.pixelManipulation = pixelManipulation;
+      inputForNextStep.lodash = lodash;
```

Comment:
Do you imagine this "toolkit" we supply for in-browser usage would grow over time, or we'd keep it really strictly limited?

Reply:
@jywarren We can add support for more stuff as we go for sure, just like adding more features to a system